Makefile

```diff
@@ -98,6 +98,10 @@ fmt:
 vet:
 	go vet ./...
 
+# Run golangci-lint
+lint:
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.0
+	golangci-lint --timeout 30m run ./...
```

Comment:
Nice :raised_hands:  Timeout is pretty large: our code base is rather small so shouldn't need that much in theory :thinking:  Either way, this would work, thanks for adding Makefile!

Reply:
@yorugac 
I could reduce the timeout.

Reply:
:+1: thanks. It should be fine with 5m, and if not, we can always tinker with it.